config/routes.rb

```diff
@@ -5,6 +5,7 @@
     get "/subscriber-lists", to: "subscriber_lists#index"
     get "/subscriber-lists/:slug", to: "subscriber_lists#show"
     post "/subscriber-lists/:slug/bulk-unsubscribe", to: "subscriber_lists#bulk_unsubscribe"
+    patch "/subscriber-lists/:slug", to: "subscriber_lists#update"
```

Comment:
Nitpick: I think this should be above the `post`, so it's next to the other `/subscriber-lists/:slug` route